how-to-test-locally/Taskfile.yaml

```diff
@@ -0,0 +1,49 @@
+version: '3'
+
+env:
+    # Directory to place the dependencies - just internal to this taskfile
+    DIR: dependencies
+
+dotenv: ['.env']
+
+tasks:
+    default:
+        cmds:
+            - task: install-requirements
+            - task: build-zip-file
+            - task: remove-dependencies-dir
+            - task: add-to-zip
+
+    install-requirements:
+        desc: "Install requirements from $REQUIREMENTS."
+        internal: true
+        requires:
+            var: REQUIREMENTS
+        cmds:
+            - rm -rf $DIR
+            - for:
+                var: REQUIREMENTS
+                split: ','
+              cmd: pip3.9 install -r ../{{ .ITEM }} -t $DIR
```

Comment:
Consider taking this from mypy.ini and python_version = 3.9 variable

In my case this was failing in my local tests (because did not have versin 3.9)

Reply:
While building it locally I see in lamda
```
[ERROR] Runtime.ImportModuleError: Unable to import module 'main_aws': No module named 'orjson.orjson'
Traceback (most recent call last):
```

Reply:
I used python 3.9 here because we use this for the tests and for our ESF terraform files:
https://github.com/elastic/elastic-serverless-forwarder/blob/30c41f4547aaf09080bd7babacb20f34c6b2cc03/.github/workflows/test.yml#L41

There is an open issue to offer support to version 3.12 as well: https://github.com/elastic/elastic-serverless-forwarder/issues/656.